pgo/cmd/auth.go

```diff
@@ -96,9 +100,9 @@ func parseCredentials(dat string) msgs.BasicAuthCredentials {
 	return creds
 }
 
-func GetCredentialsFromFile() msgs.BasicAuthCredentials {
+func getCredentialsFromFile() msgs.BasicAuthCredentials {
```

Comment:
Great change! Please document this function if we're going to be renaming it ;-)

Reply:
Function is now documented.